spring-grpc-spring-boot-autoconfigure/src/main/java/org/springframework/grpc/autoconfigure/server/GrpcServerProperties.java

```diff
@@ -65,19 +67,35 @@ public class GrpcServerProperties {
 
 	private final KeepAlive keepAlive = new KeepAlive();
 
+	private String address;
```

Comment:
Do you mind adding a comment here - this will show up in the generated config props.

This `host and port OR address` reminds me of [MongoProperties](https://github.com/spring-projects/spring-boot/blob/37ae78ea73e043dbe8b605c586e3cc845a9d2240/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/mongo/MongoProperties.java#L72). I like the idea of `deriveAddress` and leaving the properties alone `host,port,address`. I do ❤️ the validation that you can't set one and then the other (cake and eat it too) that you have done.